### DISCUSSION TITLE
When setting UI timezone, the times are incorrect

### DISCUSSION THREAD

**Discussion by KrunchMuffin** [apache/airflow#41858] [Q&A]
I have these set in the cfg
`default_timezone = UTC`
`default_ui_timezone = America/Toronto`

I ran a dag at 9.26 AM EST
It shows it ran at 5:30 and UTC was 9:30, but the clock is correct in the upper right. I had a 4 min timeout, so I guess that's why the 9.30 even though it says Run, its more like the End.

At any rate, do I have something set incorrectly? The server is also UTC.

![image](https://github.com/user-attachments/assets/dd0d7a85-437a-4830-ab45-d2c387130390)

**Comment by KrunchMuffin** [comment]
anyone?

**Reply by portorealasset** [reply]
same problem here! 
no one will show up, right
Thanks GPT :(

**Comment by potiuk** [comment]
This is quite old version of Airlfow probably and if you look at the issues and release notes you will likely find that some of the timezone issues were used. Rather than GPT, I recommend searching through past issues and release notes, and generally I strongly recommend to upgrade to latest Airflow 3 released (we are just voting on 3.1.7 - you can use the opportunity to use the latest version and test if things are working and report back- -see https://github.com/apache/airflow/issues/61255

**Reply by Andrade020** [reply]
Thank you so much!
